marimo/_config/config.py

```diff
@@ -227,6 +227,18 @@ class GoogleAiConfig(TypedDict, total=False):
     api_key: str
 
 
+@dataclass
+class SnippetsConfig(TypedDict):
+    """Configuration for snippets.
+
+    **Keys.**
+
+    - `custom_path`: the path to the custom snippets directory
+    """
+
+    custom_path: NotRequired[str]
```

Comment:
Thoughts about this being a list of strings? I could see someone asking for multiple paths

Reply:
I was planning to ask you the same question - I think a list makes more sense here. Also, should the default path optional as well in the config? Something like

```
[tools.marimo.snippets]
use_default_path: true
custom_paths:
  - foo/bar
  - foo/baz
```

Reply:
yea thats not a bad idea. maybe `include_default_snippets` with default `true`?

Reply:
Updated no as a list of strings, with `include_default_snippets` as an option with default `true`